### DISCUSSION TITLE
What do the symbols mean in the Quarto shortcuts help page?

### DISCUSSION THREAD

**Discussion by lfmcmillan** [quarto-dev/quarto-cli#10430] [Q&A] [answered]
### Description

I am asking about the Quarto shortcuts help page: https://quarto.org/docs/visual-editor/options.html

It shows various symbols for the shortcuts, e.g. something like "^ B" as a shortcut for bold, and some weird key symbol for Headings. 

What do those symbols mean in terms of keys? What keys are they on e.g. Windows?

I know the up arrow is Shift, but what are the rest of the symbols? 

Thanks

**Comment by mcanouil** [comment, marked as answer]
Thos are the Apple Mac keystrokes. <kbd>^</kbd> usually is <kbd>Ctrl</kbd>.
Keyboard shortcuts are all available as hover in the Visual Editor
![image](https://github.com/user-attachments/assets/39ff0907-621b-4755-8083-8262a3cad947)
![image](https://github.com/user-attachments/assets/01a226f2-97cf-4d9a-94d6-b79299509f40)

**Reply by lfmcmillan** [reply]
Thank you! 
Since a sizeable proportion of the population do not use Apple Macs, it would be more helpful if the help pages displayed the typical key strokes for other systems as well.

**Reply by mcanouil** [reply]
Actually, the documentation page changes depending on the OS but some symbols are still used.
On mac, `^` is `control` and the JavaScript kept the symbol for Windows `ctrl`.

I've opened an issue to fix this in the documentation.

**Comment by lfmcmillan** [comment]
Ahh ok, thanks for reporting that. The symbol for Alt was also not being
switched to Windows correctly.

Louise
